frontend/src/testDefinition.js

```diff
@@ -2,8 +2,14 @@
 export const TEST_DEFINITION = {
   emib: {
     // public test
-    sampleTest: "emibSampleTest",
+    sampleTest: {
```

Comment:
I don't think these test definitions should be modified. A test's public nature should be determined by the database and not the frontend because there are security issues there, and if we changed if we'd have to change it in multiple places.

Reply:
Undo restructure of test definition.
See commit #https://github.com/code-for-canada/project-thundercat/pull/461/commits/a99e2df5c34cef7f555d4f7b8a9734e817cf5a9d